stylesheets/style.scss

```diff
@@ -43,6 +43,21 @@ body {
 .driver-index {
   margin-top: 60px;
 
+  a {
+    margin-right: 10px;
+  }
+
+  .destroy-action {
+    display: inline-block;
+    margin-left: 30px;
+  }
+
+  .destroy-link {
+    color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.